react-common/components/controls/EmbedVideo.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { classList, ControlProps } from "../util";
+
+export interface EmbedVideoProps extends ControlProps {
+  src: "youtube" | "streams";
+  id: string;
+  autoplay?: boolean;
+  allowFullScreen?: boolean;
+//   encryptedmedia?: boolean;
+
+}
+
+
+
+export const EmbedVideo = (props: EmbedVideoProps) => {
+    const {
+       src,
+       id,
+       autoplay,
+       allowFullScreen
+
+    } = props;
+
+
+
+    let videoURL = `https://www.youtube.com/embed/${id}?${autoplay ? "autoplay=1" : ""}`;
+    let streamURL= `https://msit.microsoftstream.com/video/${id}`;
+
+    if (src=="streams"){
+        videoURL=streamURL;
+    }
+
+    let appending="";
+
+    if (autoplay){
+        appending="; autoplay";
+    }
+
+    let st1="encrypted-media; picture-in-picture";
+
+    let st2=st1+appending;
+
+    return (
+        <div className={classList("common-embed-video-wrapper")}>
+           <iframe src={videoURL} title="YouTube video player" 
```

Comment:
youtube video iframe tend to be very heavy and we have an optimization in our docs that load the screenshot by default and only mount the iframe when the user decides to play.

Reply:
sounds good, will do, but will probably end up in next pull request!